.pipelines/java_packaging.yml

```diff
@@ -0,0 +1,37 @@
+parameters:
+- name: JavaPackaging
+  displayName: Package and Publish Java Package
+  type: boolean
+  default: true
+stages:
+- stage: Package and Publish Java Package
+  jobs:
+  - job:
+    workspace:
+      clean: all
+    pool: 'onnxruntime-Win-CPU-2022'
+
+    steps:
+    - script: |
+        ./build.sh -DOCOS_BUILD_JAVA=ON
```

Comment:
how about other platforms?
It looks like it runs on Windows, while a bash script was called here.

Reply:
yep, this is just an inital implementation so i can potentially try the pipeline out on a fork - will add the other platforms for final PR.